src/test/java/com/wikia/webdriver/testcases/adstests/mobileadstests/TestDfpParamsPresentMobile.java

```diff
@@ -13,26 +14,43 @@
  * @link https://www.google.com/dfp/5441#delivery/LineItemDetail/LINE_ITEM_ID=115974612
  * @ownership AdEngineering
  */
-public class TestDfpParamsPresentMobile extends TemplateNoFirstLoad {
```

Comment:
He is no longer working in Wikia, so I'd say: yes, you can 
We should also not include any author names in the code for the same reason